src/components/Home/summary.tsx

```diff
@@ -1,7 +1,21 @@
 import Statistics from '@/components/Home/statistics'
-import { Stat } from '@/types/types'
+import useUser from '@/hooks/user'
+
+function Summary() {
+  const { data: currentUser } = useUser()
+
+  let numVisits = 0
+  let numHours = 0
+  let commutedDist = 0
+  let walkedDist = 0
+
+  if (currentUser && currentUser.stats) {
```

Comment:
You can simplify this even further by using the stats attributes directly rather than instantiating additional variables. TS will get annoyed if `currentUser` is undefined so as long as you check for that you're good.

```typescript
function Summary() {
  const { data: currentUser, isSuccess } = useUser()

  if (!isSuccess || currentUser === undefined) return null

  const {
    stats: { numVisits, numHours, commutedDist, walkedDist }
  } = currentUser

  return (...)
```

Reply:
Added extra `currentUser.stats == undefined` to avoid undefined stats error